homeassistant/components/webhook/strings.json

```diff
@@ -1,8 +1 @@
-{
-  "issues": {
-    "trigger_missing_local_only": {
-      "title": "Update webhook trigger: {webhook_id}",
-      "description": "A choice needs to be made about whether the {webhook_id} webhook automation trigger is accessible from the internet. [Edit the automation]({edit}) \"{automation_name}\", (`{entity_id}`) and click the gear icon beside the Webhook ID to choose a value for 'Only accessible from the local network'"
-    }
-  }
-}
+{}
```

Comment:
Is this better than just deleting the file? (Okay, for git history purposes, I think)

Reply:
Is indeed better to delete it completely